### PR TITLE
Feature/Add Header to RequestDVORenderer

### PR DESCRIPTION
**Pull request by Siolto**

### CODE REVIEW

File: apps/enmeshed/lib/account/mailbox/message_detail_screen.dart [hunk 1]
```diff
@@ -199,18 +199,16 @@ class _RequestInformationState extends State<_RequestInformation> {
...
+      child: RequestDVORenderer(
+        accountId: widget.account.id,
+        requestId: widget.message.request.id,
+        isIncoming: !widget.message.request.isOwn,
+        requestDVO: useRequestFromMessage ? widget.message.request : null,
+        acceptRequestText: context.l10n.accept,
+        showHeader: false,
+        onAfterAccept: () => setState(() {
+          useRequestFromMessage = false;
+        }),
```
Comment (jkoenig134) [inline apps/enmeshed/lib/account/mailbox/message_detail_screen.dart:211]: we are missing a trailing comma here.

Comment (jkoenig134) [reply apps/enmeshed/lib/account/mailbox/message_detail_screen.dart:211]: has also to be backported to mbr

File: apps/enmeshed/pubspec.lock [hunk 2]
```diff
@@ -325,14 +325,13 @@ packages:
...
     version: "1.1.1"
   flutter_inappwebview_ios:
-    dependency: "direct overridden"
-    description:
-      path: flutter_inappwebview_ios
-      ref: "27fd339172572fe1b16cdcdae235b2a88f432e3c"
-      resolved-ref: "27fd339172572fe1b16cdcdae235b2a88f432e3c"
-      url: "https://github.com/pichillilorenzo/flutter_inappwebview.git"
-    source: git
-    version: "1.0.14"
```
Comment (jkoenig134) [inline apps/enmeshed/pubspec.lock:335]: I think something went wrong here. this should still be overridden.

Comment (jkoenig134) [reply apps/enmeshed/pubspec.lock:335]: please do a flutter pub get (or flutter pub upgrade) again in this folder

Comment (Siolto) [reply apps/enmeshed/pubspec.lock:335]: interesting... somehow melos fixed it, but not flutter pub upgrade

**Review by jkoenig134 — APPROVED** [review]

File: apps/enmeshed/lib/account/mailbox/message_detail_screen.dart [hunk 3]
```diff
@@ -199,17 +199,17 @@ class _RequestInformationState extends State<_RequestInformation> {
...
+        isIncoming: !widget.message.request.isOwn,
+        requestDVO: useRequestFromMessage ? widget.message.request : null,
+        acceptRequestText: context.l10n.accept,
+        showHeader: false,
+        onAfterAccept: () => setState(
+          () {
             useRequestFromMessage = false;
-          }),
+          },
         ),
```
Comment (jkoenig134) [inline apps/enmeshed/lib/account/mailbox/message_detail_screen.dart:213]: please inline it instead

```
onAfterAccept: () => setState(() => useRequestFromMessage = false),
```

**Review by jkoenig134 — APPROVED** [review]

**Pull request merged**